requirements/text_test.txt

```diff
@@ -5,5 +5,5 @@ jiwer >=2.3.0, <=3.0.2
 rouge-score >0.1.0, <=0.1.2
 bert_score ==0.3.13
 transformers >4.4.0, <4.30.3
-huggingface-hub <0.16  # hotfix, failing SDR for latest PT 1.11
+huggingface-hub <0.17  # hotfix, failing SDR for latest PT 1.11
```

Comment:
Is the comment still true?

Reply:
```suggestion
huggingface-hub >0.16, 0.17
```

Reply:
```suggestion
huggingface-hub >0.16.0, <0.17
```
seems to be passing so let's try to freeze to the latest as it is just for testing